ci/tests/puppeteer/scenarios/single-logout-cas-frontchannel/init.sh

```diff
@@ -1,23 +1,39 @@
 echo "Starting CAS client application"
 
-if [[ ! -f /tmp/bootiful-cas-client.jar ]] ; then
+KEYSTORE=${CAS_KEYSTORE:-../../overlay/thekeystore}
+
+if [[ -z $TMPDIR ]]; then
+  TMPDIR=${TEMP:-/tmp}
+fi
+
+if [[ ! -f ${TMPDIR}/bootiful-cas-client.jar ]] ; then
   echo "Downloading Bootiful CAS client..."
-  (cd /tmp && curl -OL https://github.com/apereo/bootiful-cas-client/releases/download/v1.4.0/bootiful-cas-client.jar)
+  curl -L https://github.com/apereo/bootiful-cas-client/releases/download/v1.4.0/bootiful-cas-client.jar -o ${TMPDIR}/bootiful-cas-client.jar
 fi
 
-cat > /tmp/sslConfig.properties <<- EOM
+cat > ${TMPDIR}/sslConfig.properties <<- EOM
 ignoreSslFailures=true
 EOM
 
-cat /tmp/sslConfig.properties
-java -jar /tmp/bootiful-cas-client.jar \
-  --server.ssl.key-store="${CAS_KEYSTORE}" \
-  --cas.ssl-config-file=/tmp/sslConfig.properties &
+cat ${TMPDIR}/sslConfig.properties
+java -jar ${TMPDIR}/bootiful-cas-client.jar \
+  --management.endpoints.web.exposure.include=* \
+  --management.endpoint.shutdown.enabled=true \
+  --endpoints.shutdown.enabled=true \
+  --server.ssl.key-store="${KEYSTORE}" \
+  --cas.ssl-config-file=${TMPDIR}/sslConfig.properties &
```

Comment:
Running this with shutdown actuator enabled b/c reliably killing processes in cross platform way, from node, seems to be difficult.